dora/tests/testcontainers/src/test/java/alluxio/membership/MembershipManagerTest.java

```diff
@@ -367,13 +367,13 @@ public void testSameWorkerIdentityConflict() throws Exception {
         .setAddress(new WorkerNetAddress()
             .setHost("worker1").setContainerHost("containerhostname1")
             .setRpcPort(1000).setDataPort(1001).setWebPort(1011)
-            .setDomainSocketPath("/var/lib/domain.sock"));
+            .setDomainSocketPath("/var/lib/domain.sock").setHttpServerPort(1021));
     WorkerInfo wkr2 = new WorkerInfo()
         .setIdentity(workerIdentity1)
         .setAddress(new WorkerNetAddress()
             .setHost("worker2").setContainerHost("containerhostname2")
             .setRpcPort(2000).setDataPort(2001).setWebPort(2011)
-            .setDomainSocketPath("/var/lib/domain.sock"));
+            .setDomainSocketPath("/var/lib/domain.sock").setHttpServerPort(2021));
     membershipManager.join(wkr1);
     // bring wrk1 down and join wrk2 with a same worker identity.
     membershipManager.stopHeartBeat(wkr1);
```

Comment:
can u add a test to have a worker join without http port set and afterwards join with http port set , it should be allowed to join and the list of workers should be unchanged.

Reply:
Added a test `testWorkerHttpServerPorts`. Could you take a review?